main.go

```diff
@@ -151,6 +165,7 @@ func parseConfiguration(log *zap.SugaredLogger) Config {
 	flag.BoolVar(&config.EnableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	flag.BoolVar(&config.Version, "version", false, "Display current version and exit")
```

Comment:
not sure I understand the value of this flag. Can we just always print the version on the Operator start?